lab2.js

```diff
@@ -86,33 +106,45 @@ var hello = {
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
 
-function SentientBeing () {
-  // TODO: specify a home planet and a language
-  // you'll need to add parameters to this constructor
+function SentientBeing(homePlanet, speech) {
+
+  this.homePlanet = homePlanet;
+  this.speech = speech;
 }
 
-// sb is a SentientBeing object
-function sayHello (sb) {
-    // TODO: say hello prints out (console.log's) hello in the
-    // language of the speaker, but returns it in the language
-    // of the listener (the sb parameter above).
-    // use the 'hello' object at the beginning of this exercise
-    // to do the translating
+SentientBeing.prototype.sayHello = function(SentientBeing) {
+
+  if (SentientBeing.speech === 'klingon') {
```

Comment:
cool. this if if/else block totally works. Next challenge: figure out how to do it in two lines, referencing the hello object, using bracket notation. :)